docs/src/whatsnew/latest.rst

```diff
@@ -32,14 +32,15 @@ This document explains the changes made to Iris for this release
 ===========
 
 #. `@ESadek-MO`_ edited :func:`~iris.io.expand_filespecs` to allow expansion of
-    non-existing paths, and added expansion functionality to :func:`~iris.io.save`.
-    (:issue:`4772`, :pull:`4913`)
+   non-existing paths, and added expansion functionality to :func:`~iris.io.save`.
+   (:issue:`4772`, :pull:`4913`)
```

Comment:
This isn't currently rendering quite right. https://scitools-iris.readthedocs.io/en/latest/whatsnew/latest.html#features

Reply:
Oh that's why! @ESadek-MO and I couldn't figure it out!